wiki/Installation-guide.md

```diff
@@ -101,6 +101,9 @@ You can get appimage for latest stable version from [here](https://github.com/al
 * Download `alvr_streamer_linux.tar.gz` from the release [download page](https://github.com/alvr-org/ALVR/releases/latest).
 * Run `bin/alvr_dashboard`
 
+#### Flatpak
+For Flatpak users, refer to the instructions [here](https://github.com/alvr-org/ALVR/wiki/Flatpak)
+
```

Comment:
Move this just below `Appimage` and above `Portable tar.gz`. I believe Flatpack is still a better choice than the tar.gz.